### ISSUE TITLE
Div with absolute position is hidden in full screen mode

### ISSUE THREAD

**Issue by snubian** [brunob/leaflet.fullscreen#52]
Firstly, thank you for developing this control, it's great and incredibly easy to get working.

I am having one small problem that maybe you are aware of, and know a solution for. I have a div that uses `position: absolute` which acts as a toolbar and is positioned at the top centre of the map. This is *not* a Leaflet control, just a plain div.

When I toggle to fullscreen this div disappears. I can see that it is positioned in the correct place, but is under the map. I've tried increasing the `z-index` of the div to 100000 (i.e. one higher than `leaflet-pseudo-fullscreen` which is 99999), but to no avail.

See my example [here](http://stuartallen.com.au/refugia/map/acacia_loderi).

If you're aware of a simple fix for this I'd love to know about it. Thanks again!

**Comment by brunob** [comment]
You should put your toolbar in control panes in order to get this one "included" in your map, this will fix your bug ;)

http://leafletjs.com/reference.html#map-panes

I'm closing the issue, but feel free to reopen if needed.

**Comment by snubian** [comment]
Thanks, I was just hoping you'd know of an easy CSS work-around :)

I did originally have the toolbar in a Leaflet control but was having some difficulty getting it centre aligned in all browsers. Thanks again for your efforts in developing the plugin.

**Comment by adv0r** [comment]
hi @brunob , the link to the documentation of map-panes is now broken. I tried looking at http://leafletjs.com/reference-1.3.0.html#map-pane  and some working example but I do not understand how to add custom html inside a pane.

**Comment by prasanjitdash** [comment]
> 
> 
> You should put your toolbar in control panes in order to get this one "included" in your map, this will fix your bug ;)
> 
> http://leafletjs.com/reference.html#map-panes
> 
> I'm closing the issue, but feel free to reopen if needed.

@brunob , Hi, do you have a follow-up or a little example of this one. I have the same issue. I am using a `div `with `id='toolGroup'` to collect all plugin controls.

```
<div class="tool-group bg-light" id="toolGroup">
 // insert all tool buttons here
</div>
```
This `div` opens by clicking on another button on the map.

But, in full-screen mode it does not show up and does not respond to any CSS-based  zIndex setting. I understood that you would have to use something like createPane, but an example would be useful. That is, how to use createPane on a div (here with ID `toolGroup`). Thanks

**Comment by brunob** [comment]
@prasanjitdash a "simple" search of this method on github should be useful cf https://github.com/search?l=JavaScript&q=map.createPane&type=Code

**Comment by prasanjitdash** [comment]
Thanks, @brunob, but I think I am missing something or could not explain the issue properly. The one you linked to, I have used it before for basemap and overlay maps, as follows:
```
map.createPane('basemapcontainer');
map.getPane('basemapcontainer').style.zIndex = 199;
map.createPane('overlaymapcontainer');
map.getPane('overlaymapcontainer').style.zIndex = 200;
```
But my question is, if I have an external (to leaflet map), how do I apply this approach on the external div, so that it will remain visible in full-screen mode. The external div could be:
```
<div class="tool-group bg-light" id="toolGroup">
 // insert all tool buttons here
</div>
```
Is it something like:
`map.getPane('toolGroup').style.zIndex = 100000; //(one more that the fullScreen zIndex)
` ?

**Comment by navidmadannezhad** [comment]
@prasanjitdash Hi! did you find any solution to this problem? I have the same problem, trying to put a div with "details_modal" id on the fullscreen map, but there is no guide for putting custom html element on the map, at least I haven't find any yet :-)

**Comment by prasanjitdash** [comment]
I'm not sure about the question as I have lost context (its been a while). But my final implementation was here (last year):
https://www.star.nesdis.noaa.gov/socd/ov/

Here, there is an additional panel on top-right that holds all the controls. If you take a look and let me know what exactly you mean, that may help me to recall what I did. Thanks

**Comment by navidmadannezhad** [comment]
@prasanjitdash Thanks a lot for your response. The problem was about putting a custom html element above the leaflet map. 
I finally solved it by giving `z-index: 9999` to my element.
I also checked your implementation and you have done the same thing, giving the `z-index: 2000;` to your side-bar element.

Seems it is the only way we can put elements above the leaflet map and there is no other way related to leaflet javascript config.
